ui/apps/platform/src/Containers/Clusters/ClustersTablePanel.tsx

```diff
@@ -78,6 +79,7 @@ function ClustersTablePanel({
     const isMoveInitBundlesEnabled = isFeatureFlagEnabled('ROX_MOVE_INIT_BUNDLES_UI');
 
     const [isInstallMenuOpen, setIsInstallMenuOpen] = useState(false);
+    const [isModalOpen, setIsModalOpen] = useState(false);
```

Comment:
Several of the UI engineers have made `useModal` type hooks for their features; it may be worth standardizing that during tech debt sprint.

Example, https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/hooks/useModal.ts

Reply:
Encapsulating a hook in a hook for open and close functions versus one set function seems over abstract.